frontend/src/EDR/index.tsx

```diff
@@ -100,7 +102,7 @@ export const EDR: React.FC<Props> = ({playSoundNotification}) => {
             getTrains(serverCode).then(setTrains);
         }, 10000);
         if (!window.trainsRefreshWebWorkerId) {
-            alert(t("app.fatal_error"));
+            enqueueSnackbar(t('app.fatal_error'), { preventDuplicate: true, variant: 'error', autoHideDuration: 10000 });
```

Comment:
I'll merge because it's quite great already!
But yes, the idea was more to add it in the catch too
```tsx
            getTrains(serverCode).then(setTrains).catch(() => sendToast("A refresh was skipped")
```
I think it would be great to add it in the "fetchAllDatas" catch too, wdyt ? (before the fetching is retried 5s later)
```tsx
            }).catch(() => setTimeout(fetchAllDatas, 5000));
        }).catch(() => setTimeout(fetchAllDatas, 5000));
```

Reply:
Sure, seems appropriate. I'll add these soon.